src/components/ContactsList/ContactsListOrSpinner.spec.jsx

```diff
@@ -0,0 +1,62 @@
+import React from 'react'
+import { mount } from 'enzyme'
+
+import AppLike from '../../tests/Applike'
+
+import ContactsListOrSpinner from './ContactsListOrSpinner'
+import renderer from 'react-test-renderer'
+describe('ContactsListOrSpinner', () => {
+  test('should display the loader', () => {
+    const contact = { email: [{ address: 'johndoe@localhost' }] }
+    const contactRowInstance = (
+      <AppLike>
+        <ContactsListOrSpinner
+          contacts={[contact]}
+          hasMore={false}
+          fetchStatus={'loading'}
```

Comment:
`fetchStatus="loading"` (strange that eslint/prettier didn't catch this?)

Reply:
Why that? I really like this syntax more coherent with the rest